integration-tests/test_e2e.py

```diff
@@ -0,0 +1,37 @@
+"""
+This Python module contains integration tests for rhc.
+It uses pytest-client-tools Python module.More information about this
+module could be found: https://github.com/ptoscano/pytest-client-tools/
+"""
+
+import contextlib
+import pytest
+from time import sleep
+from utils import (
+    yggdrasil_service_is_active,
+    prepare_args_for_connect,
+)
+
+
+@pytest.mark.skip("Test cannot be run due to unresolved issues CCT-862 and CCT-696")
```

Comment:
I think it would be a good move to skip this tests till  the related issue are fixed. I will keep a note in the CCT card to re enable this test. This will help us close the test migrations. @jirihnidek